doc/user/content/releases/_index.md

```diff
@@ -30,6 +30,7 @@ Region        | Day of week | Time
 --------------|-------------|-----------------------------
 aws/eu-west-1 | Wednesday   | 2100-2300 [Europe/Dublin]
 aws/us-east-1 | Thursday    | 0500-0700 [America/New_York]
+aws/us-west-2 | Thursday    | 0200-0400 [America/Los_Angeles]
```

Comment:
Ah, this should reflect the time of the calendar event, not the local time at which the event occurs. I put up https://github.com/MaterializeInc/materialize/pull/22566 to adjust.